sources/quest_2/chapter_2/step_10/step_10.md

```diff
@@ -18,5 +18,15 @@ fun test(arg1: u64, arg2: 64) {
 
 Here `arg1` of type `u64` and `arg1` of type `u64` are the two parameters declared as a part of the function.
 
+
+🤔 You might have noticed the variable `house` is written as `_house` . Do you know why?
+
+**Note:** While you try to compile the newly created function you will recieve a warning saying `Unused assignment or binding for local 'house'. Consider removing, replacing with '_', or prefixing with '_' (e.g., '_house')` 
+
+Concept : Using `_` acknowledges that the variable is intentionally not being used. We will talk more about this in the later chapter once we start dealing with local variables.
+
+
```

Comment:
When the student reads this sentence, he doesn't know that the variable is `_house`, as he hasn't yet developed this, he only has the `start` and not the `target`.

Instead, it should be stated in the exercise description, like, 

"If a variable is not used, it must be named with a `_` suffix to avoid compilation errors. Keep this in mind, it might help you with the exercise that's coming up..."

Reply:
Done!